.circleci/config.yml

```diff
@@ -12,6 +12,7 @@ workflows:
     jobs:
       - path-filtering/filter:
           name: check-for-changes
+          tag: '3.13'
```

Comment:
odd change this - looks like circleci have broken something recently and can't get the project to build using python 3.8 any more.